esmvaltool/cmorizers/obs/cmorize_obs_clara_avhrr.ncl

```diff
@@ -0,0 +1,235 @@
+loadscript(getenv("esmvaltool_root") + \
+           "/esmvaltool/cmorizers/obs/interface.ncl")
+
+begin
+
+  ; Script name (for logger)
+  DIAG_SCRIPT = "cmorize_obs_clara_avhrr.ncl"
+
+  ; Source name
+  OBSNAME = "CLARA-AVHRR"
+
+  ; Tier
+  TIER = 3
+
+  ; Period
+  YEAR1 = 1982
+  YEAR2 = 2018
+
+  ; Selected variable (standard name)
+  VAR = (/"clt", "clivi", "clwvi"/)
+
+  ; Name in the raw data
+  NAME = (/"cfc", "iwp_allsky", "lwp_allsky"/)
+
+  ; Filename base
+  FNBASE = (/"CFCmm", "IWPmm", "LWPmm"/)
+
+  ; Conversion factor
+  ; Remark: total cloud cover (CFC) is reported as "1" but is actually "%"
+  ; IWP and LWP use scale_factor to convert to kg/m2
+  ; CONV = (/1., 1., 1./)
+
+  ; MIP
+  MIP = (/"Amon", "Amon", "Amon"/)
+
+  ; Frequency
+  FREQ = (/"mon", "mon", "mon"/)
+
+  ; CMOR table
+  CMOR_TABLE = getenv("cmor_tables") + \
+    (/"/cmip5/Tables/CMIP5_Amon", \
+      "/cmip5/Tables/CMIP5_Amon", \
+      "/cmip5/Tables/CMIP5_Amon"/)
+
+  ; Type
+  TYPE = "sat"
+
+  ; Version
+  VERSION = "V002_01"
+
+  ; Global attributes
+  SOURCE = "https://wui.cmsaf.eu/"
+  REF = "https://doi.org/10.5676/EUM_SAF_CM/CLARA_AVHRR/V002_01"
+  COMMENT = "The CM SAF data are owned by EUMETSAT and are available to " \
+    + "all users free of charge and with no conditions to use. If you wish " \
+    + "to use these products, EUMETSAT's copyright credit must be shown by " \
+    + "displaying the words 'Copyright (c) (2020) EUMETSAT' under/in each " \
+    + "of these SAF Products used in a project or shown in a publication " \
+    + "or website. Please follow the citation guidelines given at " \
+    + "https://doi.org/10.5676/EUM_SAF_CM/CLARA_AVHRR/V002_01 and also " \
+    + "register as a user at http://cm-saf.eumetsat.int/ to receive latest " \
+    + "information on CM SAF services and to get access to the CM SAF User " \
+    + "Help Desk."
+
+end
+
+begin
+
+  do vv = 0, dimsizes(VAR) - 1
+
+    log_info("Processing " + VAR(vv) + " (" + MIP(vv) + ")")
+
+    time = create_timec(YEAR1, YEAR2)
+    date = cd_calendar(time, 1)
+
+    ; Create timeseries
+    do yy = YEAR1, YEAR2
+
+      syear = sprinti("%i", yy)
+      do mm = 1, 12
+
+        smonth = sprinti("%0.2i", mm)
+
+        ; Read file
+        fname = systemfunc("ls " + input_dir_path + FNBASE(vv) + \
+                           syear + smonth + "01*.nc")
+
+        ; No files found
+        if (ismissing(fname)) then
+          continue
+        end if
+
+        ; Extract data
+        f = addfile(fname, "r")
+        val = f->$NAME(vv)$
+        if (isatt(val, "scale_factor")) then
+          scalefac = tofloat(val@scale_factor)
+        else
+          scalefac = 1.0
+        end if
+        if (isatt(val, "add_offset")) then
+          offset = tofloat(val@add_offset)
+        else
+          offset = 0.0
+        end if
+        xx = tofloat(val) * scalefac + offset
+        delete(val)
+
+        ; lwp is not a CMOR variable, derive as clwvi = lwp + iwp
```

Comment:
Do I interpret this comment correctly as: `clwvi` needs te be derived from non-CMOR variable `lwp` and `iwp`?

Reply:
Yes, this is correct. Liquid water path (lwp) is not a CMOR variable but only total cloud water path (clwvi) and cloud ice water path (clivi) are.